src/cmem/markdown/Markdown.tsx

```diff
@@ -132,14 +139,10 @@ export const Markdown = ({
         ...configDefault,
         ...configHtml,
         ...configTextOnly,
-        linkTarget: linkTargetName
```

Comment:
It is probably not correct to remove this handler. Local anchors will stay in the window but extern links are opened in new tab/window. This does not work anymore please compare storybook of last release and this feature branch.